google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/apis/CloudApiManagementConfirmationDialog.java

```diff
@@ -143,6 +180,11 @@ private ActionListener newServiceAccountClickHandler() {
     return e -> serviceAccountDetailsPane.setVisible(((JCheckBox) e.getSource()).isSelected());
   }
 
+  private String getServiceKeyPathSelectorDefaultText() {
```

Comment:
Nit: it feels from this method name like we are getting just a text , although it's a default path, maybe something like `getDefaultServiceKeyPath()`?